src/datatransfer.js

```diff
@@ -42,4 +50,26 @@ export default class DataTransfer {
 	setData( type, data ) {
 		this._native.setData( type, data );
 	}
+
+	*_getFiles() {
```

Comment:
That's because of the historical reasons. There were no `files` at the very beginning because there are some performance problems with reading files handlers, but then I realized that you need to get files handlers anyway to check if there are files, so the property was created. That's true that there is not a reason to keep it as a generator.